proto/campaign/genesis.proto

```diff
@@ -2,10 +2,13 @@ syntax = "proto3";
 package tendermint.spn.campaign;
 
 // this line is used by starport scaffolding # genesis/proto/import
+import "gogoproto/gogo.proto";
+import "campaign/campaign_chains.proto";
 
 option go_package = "github.com/tendermint/spn/x/campaign/types";
 
 // GenesisState defines the campaign module's genesis state.
 message GenesisState {
     // this line is used by starport scaffolding # genesis/proto/state
+		repeated CampaignChains campaignChainsList = 1 [(gogoproto.nullable) = false]; // this line is used by starport scaffolding # genesis/proto/stateField
```

Comment:
```suggestion
    repeated CampaignChains campaignChainsList = 1 [(gogoproto.nullable) = false]; // this line is used by starport scaffolding # genesis/proto/stateField
```

Reply:
Seems like we have some inconsistencies in generated proto files: https://github.com/tendermint/starport/issues/1561